apps/server/src/terrain/communication/simconnect.ts

```diff
@@ -251,12 +286,28 @@ export class SimConnect {
         }
     }
 
+    private simConnectSystemEvent(message: SystemEventMessage): void {
+        if (message.eventId === SystemEventId.SimulatorState) {
+            const sim = message.content as SystemEventSim;
+            if (sim.running === false && this.callbacks.reset !== null) {
+                this.callbacks.reset();
+            }
+        } else if (message.eventId === SystemEventId.PauseState) {
+            const pause = message.content as SystemEventPause;
+            if (pause.type !== SystemEventPauseType.Unpaused) {
+                if (this.callbacks.paused !== null) this.callbacks.paused();
+            } else if (this.callbacks.unpaused !== null) this.callbacks.unpaused();
+        } else {
+            this.logging.error(`Unknown system event ID: ${message.eventId}`);
+        }
+    }
+
     private connectToSim() {
         if (this.shutdown) return;
 
         this.connection = new Connection();
         if (this.connection.open(SimConnectClientName) === false) {
-            this.logging.error(`Connection failed: ${this.connection.lastError()} - Retry in 10 seconds`);
+            this.logging.error(`Connection to MSFS failed: ${this.connection.lastError()} - Retry in 10 seconds`);
```

Comment:
Could you just add a log entry for the success case when the connection is established? Some people have been a bit confused why there were errors only and I think that would make it clearer

Reply:
Fixed. Showing the error only once